### ISSUE TITLE
Server stuck restarting (docker)

### ISSUE THREAD

**Issue by bkraul** [go-gitea/gitea#17525]
### Gitea Version

e69521f

### Git Version

1.16.0

### Operating System

Ubuntu 20.04.3 (docker)

### How are you running Gitea?

Using docker dev image, but also verified the server is unavailable on https://try.gitea.io

### Database

MySQL

### Can you reproduce the bug on the Gitea demo site?

Yes

### Description

The server appears stuck on starting. This causes the web interface to not be available. This also happens on the docker image.

### Screenshots

![image](https://user-images.githubusercontent.com/14616851/139866712-ff8ce729-3be9-48ba-87bf-bb4c22d3fa27.png)

**Comment by zeripath** [comment]
Please could you give us some logs and the SHA of the version of Gitea you're using.

**Comment by bkraul** [comment]
This is all I get from the docker container stdout
```
server_1  | Received signal 15; terminating.
server_1  | Server listening on :: port 22.
server_1  | Server listening on 0.0.0.0 port 22.
server_1  | 2021/11/02 13:25:02 cmd/web.go:102:runWeb() [I] Starting Gitea on PID: 18
server_1  | 2021/11/02 13:25:02 cmd/web.go:146:runWeb() [I] Global init
server_1  | 2021/11/02 13:25:02 routers/init.go:105:GlobalInit() [I] Git Version: 2.30.2, Wire Protocol Version 2 Enabled
server_1  | 2021/11/02 13:25:02 routers/init.go:108:GlobalInit() [I] AppPath: /app/gitea/gitea
server_1  | 2021/11/02 13:25:02 routers/init.go:109:GlobalInit() [I] AppWorkPath: /app/gitea
server_1  | 2021/11/02 13:25:02 routers/init.go:110:GlobalInit() [I] Custom path: /data/gitea
server_1  | 2021/11/02 13:25:02 routers/init.go:111:GlobalInit() [I] Log path: /data/gitea/log
server_1  | 2021/11/02 13:25:02 routers/init.go:112:GlobalInit() [I] Configuration file: /data/gitea/conf/app.ini
server_1  | 2021/11/02 13:25:02 routers/init.go:113:GlobalInit() [I] Run Mode: Prod
server_1  | Received signal 15; terminating.
server_1  | Server listening on :: port 22.
server_1  | Server listening on 0.0.0.0 port 22.
server_1  | 2021/11/02 13:26:04 cmd/web.go:102:runWeb() [I] Starting Gitea on PID: 18
server_1  | 2021/11/02 13:26:04 cmd/web.go:146:runWeb() [I] Global init
server_1  | 2021/11/02 13:26:04 routers/init.go:105:GlobalInit() [I] Git Version: 2.30.2, Wire Protocol Version 2 Enabled
server_1  | 2021/11/02 13:26:04 routers/init.go:108:GlobalInit() [I] AppPath: /app/gitea/gitea
server_1  | 2021/11/02 13:26:04 routers/init.go:109:GlobalInit() [I] AppWorkPath: /app/gitea
server_1  | 2021/11/02 13:26:04 routers/init.go:110:GlobalInit() [I] Custom path: /data/gitea
server_1  | 2021/11/02 13:26:04 routers/init.go:111:GlobalInit() [I] Log path: /data/gitea/log
server_1  | 2021/11/02 13:26:04 routers/init.go:112:GlobalInit() [I] Configuration file: /data/gitea/conf/app.ini
server_1  | 2021/11/02 13:26:04 routers/init.go:113:GlobalInit() [I] Run Mode: Prod
server_1  | Received signal 15; terminating.
server_1  | Server listening on :: port 22.
server_1  | Server listening on 0.0.0.0 port 22.
server_1  | 2021/11/02 13:27:05 cmd/web.go:102:runWeb() [I] Starting Gitea on PID: 18
server_1  | 2021/11/02 13:27:05 cmd/web.go:146:runWeb() [I] Global init
server_1  | 2021/11/02 13:27:05 routers/init.go:105:GlobalInit() [I] Git Version: 2.30.2, Wire Protocol Version 2 Enabled
server_1  | 2021/11/02 13:27:05 routers/init.go:108:GlobalInit() [I] AppPath: /app/gitea/gitea
server_1  | 2021/11/02 13:27:05 routers/init.go:109:GlobalInit() [I] AppWorkPath: /app/gitea
server_1  | 2021/11/02 13:27:05 routers/init.go:110:GlobalInit() [I] Custom path: /data/gitea
server_1  | 2021/11/02 13:27:05 routers/init.go:111:GlobalInit() [I] Log path: /data/gitea/log
server_1  | 2021/11/02 13:27:05 routers/init.go:112:GlobalInit() [I] Configuration file: /data/gitea/conf/app.ini
server_1  | 2021/11/02 13:27:05 routers/init.go:113:GlobalInit() [I] Run Mode: Prod
server_1  | Received signal 15; terminating.
~/docker-compose/gitea$ docker-compose pull
Pulling db     ... done
Pulling server ... done
~/docker-compose/gitea$ docker-compose exec server bash
Error response from daemon: Container cf7997d65b6d9b93e96fc7d86e035b80152dbf517c3e957df75752369194a4e6 is restarting, wait until the container is running
```

**Comment by zeripath** [comment]
There may be something in /data/gitea/log that would be helpful

**Comment by lunny** [comment]
Fixed by #17526

**Comment by bkraul** [comment]
Updated image resolved the issue. Thanks for the fast turnaround!
